src/screens/account-details.tsx

```diff
@@ -261,22 +262,30 @@ function BalanceInput({
   }, [balance, decimals])
 
   return (
-    <Input
-      onChange={(e) => setValue(e.target.value)}
-      onClick={(e) => e.stopPropagation()}
-      onBlur={(e) => {
-        const newValue = parseUnits(e.target.value as `${number}`, decimals)
-        if (newValue !== balance) {
-          mutate({
-            address,
-            tokenAddress,
-            value: newValue,
-          })
-        }
-      }}
-      height="24px"
-      style={{ textAlign: 'right' }}
-      value={value}
-    />
+    <>
+      {loading ? (
+        <Box style={{ textAlign: 'center' }}>
```

Comment:
Wonder if we can make this spinner beside the input field (on the left), rather than hiding it?

Reply:
I can do that, it could be passed down to the input as a prop for 'pending' or 'loading' which then makes the spinner actually part of the Input component.

Reply:
I've updated it to look like this:

![Screen Shot 2023-09-15 at 11 19 33 am](https://github.com/paradigmxyz/rivet/assets/2107693/771f05b6-be3f-494d-8f24-78248a799091)